patches/tModLoader/Terraria/Item_tML.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Terraria.ModLoader.IO;
+
+namespace Terraria
+{
+	public partial class Item : TagSerializable
```

Comment:
This is good practice, we should probably employ it for other classes @JavidPack @bluemagic123